### DISCUSSION TITLE
What's or where's api-v2?

### DISCUSSION THREAD

**Discussion by albuspiroglu** [sionescu/bordeaux-threads#97] [Q&A] [answered]
Hi,
I was wishing to do a minor change in the code, then, given that this is my first time reading the bt code, I got confused with the v1 & v2 structure. My quicklisp pulled bt doesn't have any code from v2 component. Why is that? Is it some other project that maintains the quicklisp version? Should I just ignore that and download & use this on my lisps?

Thanks
Albus

**Comment by sionescu** [comment, marked as answer]
I haven't released APIv2 yet. It's on the master branch and I'd like it to receive some wider testing before releasing.

**Reply by albuspiroglu** [reply]
Before closure for this question, might I also ask whether I should try testing v2 myself? Would the simplest way be commenting out the v1 in .asd, and adding a package alias :bt & :bordeaux-threads?

**Comment by sionescu** [comment]
Yes, I encourage testing as much as possible, and I plan to make a release sometine in the next couple of months.
I intentionally made v1 and v2 loadable in parallel stoo ease with migration so I'd just rewrite all references to the "bt" package to "bt2", e.g. `sed -i 's%bt:%bt2:%g' *.lisp`

**Reply by albuspiroglu** [reply]
sounds good! thanks :)
